dev-docs/bidders/resetdigital.md

```diff
@@ -20,30 +20,37 @@ media_types: banner, video
+
+### Bid Params
 
 {: .table .table-bordered .table-striped }
 
 | Name     | Scope    | Description | Example                            | Type     |
 |----------|----------|-------------|------------------------------------|----------|
-| `pubId` | required |    Publisher account id         | `'123pubId'` | `string` |
-| `siteID` | optional |    Publisher site id         | `'123siteId'` | `string` |
+| `pubId` | required |    Publisher Account ID provided by ResetDigital    | `'123pubId'` | `string` |
+| `siteID` | optional |    Publisher Site ID         | `'123siteId'` | `string` |
 | `zoneId` | optional |   Used for extra fields          | `{}` | `object` |
+| `zoneId.placementId` | optional | ID used for reporting purposes | `"<id>"`    | `string` |
+| `zoneId.deals` | optional | Deal IDs comma-separated | `"deal123,deal456"` | `string` |
+| `zoneId.test` | optional | Flag to force bidder response with a creative | `1` | `integer` |
 | `forceBid` | optional | Returns test bid | true | `boolean` |
-| `position`     | optional           | Set the page position. Valid values are "atf" and "btf".                                                                    | `'atf'`                                                                             | `string`         |
-| `bidFloor`       | optional           | Sets the global floor -- no bids will be made under this value.                                                             | `0.50`                                                                              | `float`          |
-| `latLong`     | optional           | Sets the latitude and longitude for the visitor                                                                            | `[40.7608, 111.8910]`                                                               | `Array<float>`   |
-| `inventory`   | optional           |  This parameter allows the definition of an object defining arbitrary key-value pairs concerning the page for use in targeting. The values must be arrays of strings. | `{"rating":["5-star"], "prodtype":["tech","mobile"]}`                               | `object`         |
-| `visitor`      | optional           | This parameter allows the definition of an object defining arbitrary key-value pairs concerning the visitor for use in targeting. The values must be arrays of strings. | `{"ucat":["new"], "search":["iphone"]}`                                             | `object`         |
-| `keywords`     | optional           | This can be used to influence reports for client-side display. To get video or server-side reporting, please use First Party data or the inventory/visitor parameters. | `["travel", "tourism"]`                                                             | `Array<string>`  |
+| `position` | optional | Set the page position. Valid values are "atf" and "btf".| `'atf'`| `string`         |
```

Comment:
Position, floor and latLong need to come first from the Prebid.js standard location. It's ok to have a bidder-level override, but the documentation should say "Override the Prebid.js page position/bidFloor/etc"

Reply:
Understood, I'll make the necessary adjustments. 
Thanks for your guidance.